web/src/app/core/services/project.service.ts

```diff
@@ -243,4 +244,25 @@ export class ProjectService {
         map((projects: IProject[]) => projects.map((project: IProject) => new ProjectModel(project)))
       );
   }
+
+  /**
+   * Update the followers count in project collection
+   * @param projectUid string
+   * @param counter string
+   */
+  public updateFollowers(projectUid: string, counter: number): Observable<void> {
+    return this.activityService
+      .start()
+      .pipe(
+        switchMap(() => this.afs
+          .collection<IProject>('projects')
+          .doc<IProject>(projectUid)
+          .set(
+            {
+              followers: firestore.FieldValue.increment(counter),
```

Comment:
Only the admin/owner of the project can write/update the project document. This will have to be done another way 🤔

Reply:
When not the owner of a project, after allowing the follow/unfollow button appear for non-owners, when clicking...

<img width="2369" alt="Screenshot 2019-10-19 08 52 26" src="https://user-images.githubusercontent.com/624760/67139972-c8fdf180-f24d-11e9-9801-ce8ffd0473f1.png">

Reply:
This might need to move to a callable function. I will have a look into it 🤓